requirements_all.txt

```diff
@@ -23,6 +23,9 @@ attrs==17.4.0
 # homeassistant.components.doorbird
 DoorBirdPy==0.1.2
 
+# homeassistant.components.homekit
+HAP-python==1.1.2
```

Comment:
1.1.5 now in pypi and git - some small enhancements. There should be no breaking changes.